dataset_eval/evaluate.py

```diff
@@ -59,12 +59,17 @@ def evaluate_dataset(eval_job, dataset_dir, storage_dir):
         groundtruth_path = os.path.join(eval_location, "groundtruth.yaml")
         with open(groundtruth_path, "w") as f:
             yaml.dump(create_groundtruth_dict(snapshot["data"]["name"], train), f)
-
+        """
+        Passing more user preferences to train the model
+        """
         logging.info("Training model...")
         results = gaia_wrapper.train_model(
             project_dir=eval_location,
             groundtruth_file=groundtruth_path,
             filelist_file=filelist_path,
+            c_value = eval_job["c_value"],
```

Comment:
just a small nitpick, but no space before and after operators in function calls. see the line above this.